sdk/core/azure-core/CMakeLists.txt

```diff
@@ -96,6 +96,7 @@ set(
     inc/azure/core/internal/json/json_optional.hpp
     inc/azure/core/internal/json/json_serializable.hpp
     inc/azure/core/internal/strings.hpp
+    inc/azure/core/internal/testing_macro.hpp
```

Comment:
name it somewhat more generic, like `test_hooks.hpp`? What if in the future we want to add a non-macro there, then the name will either not be accurate, or we'll need an another header.

Reply:
I thought of that and left it since it was internal-only and we could rename it later (carefully) when the time came.

But I like your file name suggestion, so will take it :)